docroot/profiles/custom/cgov_site/modules/custom/cgov_blog/src/Plugin/Block/BlogCategories.php

```diff
@@ -72,17 +74,33 @@ public function build() {
 
     // Return blog category elements.
     $blog_categories = $this->drawBlogCategories();
+    // Get series nid for the block.
+    $series_nid = $this->blogManager->getSeriesId();
     $build = [
-      'blog_categories' => $blog_categories,
+      'blog_categories' => [
+        '#theme' => 'cgov_blocks_blog_categories',
+        '#rows' => $blog_categories,
+      ],
       '#cache' => [
         'tags' => [
-          'node_list',
+          'node:' . $series_nid,
         ],
       ],
     ];
     return $build;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function blockAccess(AccountInterface $account) {
+    $blog_categories = $this->drawBlogCategories();
+    if (count($blog_categories) > 0) {
+      return AccessResult::allowed();
+    }
+    return AccessResult::forbidden();
+  }
```

Comment:
There needs to be a cache tag added to the AccessResult. As it is not set the block will not clear under the scenario defined in https://github.com/NCIOCPL/cgov-digital-platform/pull/2346#issuecomment-524038684.